spec/mailers/user_upgrade_request_mailer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+describe UserUpgradeRequestMailer do
+  describe "sending an email to a user" do
+    let(:mail) { described_class.upgrade_request_email(user_email: "test@example.gov.uk") }
+
+    before { mail }
```

Comment:
I dont think this is needed, the first time you call `mail` in the test it will automatically call line 5 and initialise it and its value